.github/workflows/integration.yaml

```diff
@@ -2,7 +2,8 @@ name: Build
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "main", "releases/**" ]
```

Comment:
Ideally we should not create/push an image on every change to [ "main", "releases/**" ] branches since we don't have test so far. Suppose someone committed a change to main and broke the image, then no one can run the example based on that branch (ex: main). But I don't think that's a big deal. We can improve it if necessary in the future.